source/_posts/2017-05-16-shipit-day-viii-spring-2017.markdown

```diff
@@ -0,0 +1,120 @@
+
+>�Our ShipIt project was a traffic visualization system for web services supporting Millennium. In this project we used Splunk API to collect the data from services and visualize the traffic between nodes in using [Netflix�s vizceral project.](https://github.com/Netflix/vizceral)
+
+Judge Jason Heiting was impressed. �Very neat visualizations.  Intuitive UI for navigation.�
+
+Last but not least, was third place winners, Flying Mongoose (Cihan Kaynak, Matt Stramel, Mahesh Acharya, Taylor Clay).
+
+>�We created a cross platform mobile app which can be used by a pharmacy technician while picking up and scanning medications at the pharmacy to refill [Cerner�s RxStation](https://www.cerner.com/solutions/automated-dispensing-cabinet) cabinets. Prior to this project, an RCP based desktop app was running in a PC mounted on a cart with an attached barcode scanner and label printer. This type of bulky setup was reducing the mobility of the technician.�
+
+<div align="center">
+{% img /assets/2017-05-16-shipit-day-viii-spring-2017/pic11.png %}
+<sub>The Assembly was crowded with associates watching presentations.</sub>
+</div>
+
+Part of the ShipIt Day fun is livestreaming the presentations and allowing our associates from all over to participate by voting for their favorite team name, favorite presentation and favorite project. The winners of the categories are as followed�
+ 
+Favorite Presentation- _Sip It_- Patrick Gross, Adam Tibbs, Mario Perez, Robert Cornett, Trey Peek
+ 
+Favorite Team Name- _CHAPPiE_- Karthik Nimmagadda, Manidee Gattamaneni, Venkatesh Sridharan, Prasanth Chakka , Wendell Wolfe
+
+Favorite Project- _Freudian Ship_- Kate Young, Erin Dorpinghaus, Courtney Ground, Chris Farr and Sean Albery
+
+<div align="center">
+{% img /assets/2017-05-16-shipit-day-viii-spring-2017/pic12.png %}
+<sub>People�s Choice Winners with the coveted cheeseballs!</sub>
+</div>
+
+ShipIt continues to be a great way for associates to work on something they haven�t had a chance to work on, build relationships within different organizations and most importantly, enjoy themselves. Ian Kottman, an engineer on the Population Health Genomics team, has participated in multiple ShipIt Days. The reason? �I enjoy getting the time to create something that will make someone's life easier.� Carlo Filipelli, system engineer, also agrees, adding on, �ShipIt is great because it allows associates to work on interesting projects that they might not be able to get to during normal work hours. It fosters innovation and new ideas that would typically get overlooked in our busy day to day jobs. It is also a conducive environment for learning, collaboration, and having some fun!�
+
+Thank you to all our participants and a special thank you to judges Jason Heiting, Justin Morrison, Dwight Sloan and Dan Stocksick. 
+ 
+Mark your calendars- ShipIt Day IX will be held in July 2017. If you are interested in learning more about other ShipIt Days, see these earlier posts:
+<ul>
```

Comment:
You can remove the `<ul>` / `<li />` tags, and just use asterisks (as it will make the bullet list). Just make sure there is a whitespace line before beginning.